### PR TITLE
Adding Hitesh Sir's YouTube Channel

### LINKED ISSUE #2096: Want to a Youtube channelin Web Devolepment  field
### Site URL

https://www.youtube.com/@HiteshChoudharydotcom

### Description of the above provided site

It is an underrated channel for web development, android development, DevOps and a lot

### Category

Youtube

### Sub Category

Web Development 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Darkrider0007**
## Fixes Issue 
Closes #2096

## Changes proposed

- Adding Hitesh Sir's YouTube Channel
## Screenshots
image

## Note to reviewers
Thank You this is a amazing resource

### CODE REVIEW

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -358,5 +358,13 @@
...
     "subcategory": "web-development",
     "language": "hindi"
+  },
+  {
+    "name": "Hitesh Choudhary",
+    "description": "It is an underrated channel for web development, android development, DevOps and a lot",
+    "url": "https://www.youtube.com/@HiteshChoudharydotcom",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "English"
```
Comment (CBID2) [inline database/youtube/web-development.json:368]: [suggestion] This field is only for non-English sources 
```suggestion
    
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just needs a few tweaks

File: database/youtube/web-development.json [hunk 2]
```diff
@@ -358,5 +358,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "hindi"
+  },
+  {
+    "name": "Hitesh Choudhary",
+    "description": "It is an underrated channel for web development, android development, DevOps and a lot",
```
Comment (CBID2) [inline database/youtube/web-development.json:364]:
```suggestion
    "description": "It is an underrated channel for Web Development, Android development, DevOps, and more",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just gave another comment

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@Darkrider0007 
Deployment might be failing due to empty line

image

Check locally and fix it.

**Comment by Darkrider0007** [comment]
Please take a loo i fix the issue

File: database/youtube/web-development.json [hunk 3]
```diff
@@ -358,5 +358,13 @@
...
     "subcategory": "web-development",
     "language": "hindi"
+  },
+  {
+    "name": "Hitesh Choudhary",
+    "description": "It is an underrated channel for Web Development, Android development, DevOps, and more",
+    "url": "https://www.youtube.com/@HiteshChoudharydotcom",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
```
Comment (CBID2) [inline database/youtube/web-development.json:368]: [suggestion] The language feature is only for non-English resources. 
```suggestion
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Darkrider0007. Your PR is off to a good start. It just needs some more tweaking.

**Comment by Anmol-Baranwal** [comment]
@CBID2 
I found out it is not the case, even I missed it up to now. 

image

So, I don't think there is need to remove the `english` from the PR.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Darkrider0007** [comment]
@CBID2 please approve if there is not any issue

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**